include/baresip.h

```diff
@@ -333,17 +333,6 @@ struct config_avt {
 	uint32_t rtp_timeout;   /**< RTP Timeout in seconds (0=off) */
 };
 
-/** Network Configuration */
-struct config_net {
-	int af;                 /**< AF_UNSPEC, AF_INET or AF_INET6 */
-	char ifname[64];        /**< Bind to interface (optional)   */
-	struct {
-		char addr[64];
-		bool fallback;
-	} nsv[NET_MAX_NS];      /**< Configured DNS nameservers     */
-	size_t nsc;             /**< Number of DNS nameservers      */
-};
-
```

Comment:
I dont agree with this change.

This code belongs to baresip. if you need similar code in a different project, it is better
to write the code from scratch or to copy parts of it.

Reply:
Do you mean you don't want to have "struct network" in libre? I could revert this commit and leave network in baresip.
Or do you mean you don't find the module useful?
Or do you mean that you don't find request.c in libre useful at all?